rotkehlchen/tests/utils/factories.py

```diff
@@ -89,7 +89,8 @@ def make_ethereum_transaction(tx_hash: Optional[bytes] = None) -> EvmTransaction
 def make_ethereum_event(
     index: int,
     tx_hash: Optional[bytes] = None,
-    asset: Asset = A_USD,
+    # technically accepts any asset, but should be a crypto asset for ethereum events
+    asset: Asset = A_USDT,
```

Comment:
so why not change the type?

Reply:
Since constants have type `Asset`, so for having the default value, the type should be `Asset`. But perhaps we could change the default value to `None` and use better types?

(I am not iterating on the PR, just accidentally saw this comment and wanted to reply)